packages/server/src/oauth/token.test.ts

```diff
@@ -18,12 +18,27 @@ import { hashCode } from './token';
 jest.mock('jose', () => {
   const core = jest.requireActual('@medplum/core');
   const original = jest.requireActual('jose');
+  let count = 0;
```

Comment:
Test is based on counting the number of iterations to determine coverage of the different errors and invoking the jwt verify method